engine/src/main/scala/cromwell/engine/package.scala

```diff
@@ -87,5 +87,6 @@ package object engine {
                                             failureMode: WorkflowFailureMode) {
     def id = backendDescriptor.id
     def namespace = backendDescriptor.workflowNamespace
+    def name = namespace.workflow.unqualifiedName
```

Comment:
@geoffjentry Just the `EngineWorkflowDescriptor`? Most of the stuff there is from the oldestyle and might be removed eventually though.